src/omnicore/omnicore.cpp

```diff
@@ -784,6 +784,7 @@ static int parseTransaction(bool bRPConly, const CTransaction& wtx, int nBlock,
     std::vector<std::string> script_data;
     std::vector<std::string> address_data;
     std::vector<int64_t> value_data;
+    uint16_t nulldata_vout = 0;
```

Comment:
We could still discuss changing the rules later, but at least for this PR, I'd be in favor of just improving the performance. :)